tests/integration/components/image-cropper-test.js

```diff
@@ -1,26 +1,17 @@
-import { module, skip } from 'qunit';
+import { module, test } from 'qunit';
 import { setupRenderingTest } from 'ember-qunit';
 import { render } from '@ember/test-helpers';
 import { hbs } from 'ember-cli-htmlbars';
 
 module('Integration | Component | image-cropper', function (hooks) {
   setupRenderingTest(hooks);
 
-  skip('it renders', async function (assert) {
+  test('it renders', async function (assert) {
```

Comment:
Can we write tests for the image cropper as well ?

Reply:
I tried but I really can't understand what to write, also there is no documentation for testing the cropper js library

Reply:
Ok got it